### ISSUE TITLE
files with uppercase extensions not selectable or processed anymore

### ISSUE THREAD

**Issue by fritzmg** [contao/core#8317]
In Contao `3.5.11` images (files) with an uppercase extension are not visible in the file picker anymore, for instance when selecting an image for the `text` or `image` content element, or for the news teaser image of a news entry. I am guessing this happens probably due to one of the recent changes introduced by #8295 and/or #8289.

**Comment by fritzmg** [comment]
Not only are they not selectable, they aren't even processed at all anymore, if they were selected prior to the Contao update. e.g. a news entry that had a file selected as its news teaser with an uppercase extension, for example `files/test.JPG` will not show an image in newslists etc.

This happens on a CentOS 6.4 system with PHP 5.6 and Apache 2.4.

**Comment by fritzmg** [comment]
The problem is this change: f0ccac8f9ba26ecdc4bfb381e220494fb348dd5b
This removed 
```php
return strtolower($this->arrPathinfo['extension']);
```
which is causing [this](https://github.com/contao/core/blob/3.5.11/system/modules/core/library/Contao/Image.php#L119) to be false:
```php
if (!in_array($this->fileObj->extension, $arrAllowedTypes))
{
	throw new \InvalidArgumentException('Image type "' . $this->fileObj->extension . '" was not allowed to be processed');
}
```
thus causing the image not being processed in the frontend.

System log entry:
```
Image "files/test.JPG" could not be processed: Image type "JPG" was not allowed to be processed
```

**Comment by leofeyer** [comment]
Fixed the first problem in f57ca357e275b78a2c181751c0d0aaabe4e292af.

**Comment by fritzmg** [comment]
Hm, I am guessing this fixes both problems actually?

**Comment by leofeyer** [comment]
Are you just guessing or do you know? 😄

**Comment by fritzmg** [comment]
I don't know actually, sorry - didn't have time to test and I had to revert the affected Contao installation back to `3.5.9` where I had a test case on my hand. I'll may be have time tomorrow to test again.

**Comment by leofeyer** [comment]
Fixed another issue in 1f7ba063f28ffad28195b9ef935fe68119191226 and ran all tests again. This time with a file with an uppercase extension :)
